src/app/page.tsx

```diff
@@ -3,26 +3,43 @@
 import { LiveImageShapeUtil } from "@/components/live-image";
 import * as fal from "@fal-ai/serverless-client";
 import { Editor, Tldraw } from "@tldraw/tldraw";
+import { useCallback } from "react";
 
 fal.config({
   requestMiddleware: fal.withProxy({
     targetUrl: "/api/fal/proxy",
   }),
 });
 
+const shapeUtils = [LiveImageShapeUtil];
+
 export default function Home() {
   const onEditorMount = (editor: Editor) => {
+    // If there isn't a live image shape, create one
+    const liveImage = editor.getCurrentPageShapes().find((shape) => {
+      return shape.type === "live-image";
+    });
+
+    if (liveImage) {
+      return;
+    }
+
     editor.createShape({
       type: "live-image",
       x: 120,
       y: 180,
+      isLocked: true,
```

Comment:
I tried locking the live image shape, so that you can edit/erase the other shapes without interfering with your custom shape.

I think it would be helpful if tldraw provided a `canErase` flag to help out with this use-case. I'll try to add it to the tldraw library!

Reply:
This is neat! I was wondering is this was supported at all, but missed the `isLocked`